velox/functions/lib/MapConcat.cpp

```diff
@@ -20,6 +20,7 @@
 
 namespace facebook::velox::functions {
 namespace {
+static const char* kDuplicateKey = "Duplicate map keys ({}) are not allowed";
```

Comment:
This variable is used only once, do we need it?

Reply:
Removed variable